assets/js/modules/idea-hub/index.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Idea Hub module initialization.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { registerStore as registerDataStore } from './datastore/index';
+
+export const registerStore = ( registry ) => {
+	registerDataStore( registry );
+};
```

Comment:
We can just re-export the `registerStore` function like this:

```suggestion
export { registerStore } from './datastore';
```